### PR TITLE
Update politische-vierteljahresschrift.csl

### PR DESCRIPTION
**Pull request by damnation333**
via https://forums.zotero.org/discussion/64799/style-update-request-politische-vierteljahresschrift

How do I get it to show only "u." instead of "und" in the in-text citations?

### CODE REVIEW

**Comment by rmzelle** [comment]
> How do I get it to show only "u." instead of "und" in the in-text citations?

I'm not sure if CSL processors support the use of `and="symbol"` while defining a "symbol" variant of the "and" term as e.g. "u.". The spec suggests it should be hardcoded, although what I propose should validate and might work.

We should probably change the spec to just allow the "and" attribute to switch between the "long" and "symbol" forms of the "and" term (and perhaps the "short" form as well, for this case, either by adding a third attribute value, like "text-short", or by replacing the "and" attribute by an "and" child element of cs:names (with a "form" attribute to select the desired term variant), but that might be a bit awkward). (this has come up before in personal email between @adam3smith and me around 5/13/15, regarding item-locale specific settings)

File: politische-vierteljahresschrift.csl [hunk 1]
```diff
@@ -47,25 +56,30 @@
...
+            </group>
+          </if>
+        </choose>
+      </else>
     </choose>
   </macro>
-  <macro name="title">
-    <text variable="title"/>
-  </macro>
+  <macro name="title"/>
```
Comment (rmzelle) [inline politische-vierteljahresschrift.csl]: @damnation333, empty macros make @csl-bot sad.

**Comment by adam3smith** [comment]
no, sorry, citeproc-js behaves according to spec. You can't override the ampersand for and="symbol"

**Comment by rmzelle** [comment]
Remember if we have a ticket for this? (I couldn't find one in "schema" or "documentation", and couldn't dig up anything in our email conversation)

**Comment by adam3smith** [comment]
no, not that I can remember

**Comment by rmzelle** [comment]
Thanks both!

> Remember if we have a ticket for this?

I created a ticket at https://github.com/citation-style-language/schema/issues/137.

**Pull request merged**